app/assets/stylesheets/aboutus.scss

```
@@ -1,5 +1,94 @@
-.about-us {
+@import "variables";
+
+.about-us-container {
+  text-align: center;
+  padding: 3% 0;
+  color: $blue-purple;
+
+  div.au-card {
+    display: flex;
+    justify-content: space-around;
+    flex-wrap: wrap;
+  }
+
   img {
-    width: 20%;
+    width: 40%;
+  }
+
+  hr.style-five {
+    border: 0;
+    height: 0;
+    margin: 1% 5% 4%;
+    box-shadow: 0 0 5px 10px $light-blue;
+  }
+
+  hr.style-five:after {
+    content: "\00a0";
+  }
```

```suggestion
  hr {
    &.style-five {
      border: 0;
      height: 0;
      margin: 1% 5% 4%;
      box-shadow: 0 0 5px 10px $light-blue;
    }
  
    &.style-five:after {
      content: "\00a0";
    }
  }
```

Using `&` allows you to concatenate selectors